test/harness/wasm-module-builder.js

```diff
@@ -100,17 +107,23 @@ class WasmFunctionBuilder {
 
   addBody(body) {
     for (let b of body) {
-      if (typeof b !== 'number' || (b & (~0xFF)) !== 0 )
```

Comment:
Good catch. I had just blindly copied over the v8 version instead of merging the two.